autoware_map_msgs/msg/PointCloudMapMetaData.msg

```diff
@@ -0,0 +1,4 @@
+# Header
+std_msgs/Header header
+
+PointCloudMapCellMetaDataWithID[] pointcloud_map_metadata
```

Comment:
How about making this plural, or sound like plural? Something like:
```suggestion
PointCloudMapCellMetaDataWithID[] metadata_list
```

Reply:
Thanks!! 
I made this plural in https://github.com/autowarefoundation/autoware_msgs/pull/57/commits/ba347e2ee444361930691ca8a56b21fe53bf8c58